packages/SwingSet/docs/vat-upgrade.md

```diff
@@ -128,23 +135,26 @@ initializeBar(barData);
 
 ## From Outside: the AdminNode Upgrade API
 
-The upgrade must be triggered by something holding the vat's `adminNode` object. This is typically the same thing that created the vat in the first place, but a common pattern is to hand the `adminNode` to some sort of governance object. The ability to upgrade a vat is also the ability to control its behavior, and any users who expect some particular vat behavior (e.g. they auditing some contract code) must take into account the possibility of upgrade, which means they'll care about who exactly can cause an upgrade to occur.
+The upgrade must be triggered by something holding the vat's "adminNode" control facet. This is typically the same thing that created the vat in the first place, but a common pattern is to hand it to some sort of governance object. The ability to upgrade a vat is also the ability to control its behavior, and any users who expect some particular vat behavior (e.g. when auditing some contract code) must take into account the possibility of upgrade, which means they'll care about who exactly can cause an upgrade to occur.
```

Comment:
I dunno, the new "it" feels ambiguous. We've got two objects: the control facet, and the holder. Maybe:

Upgrades are triggered by invoking the `upgrade` method on the vat's "adminNode" control facet. This control facet is returned when the vat is first created (along with the new vat's root object), so the original creator is frequently the one to drive upgrades later. But a common pattern is to hand the control facet to some sort of governance object.

Reply:
Done.